src/cvodes/cvodes.c

```diff
@@ -8609,16 +8658,17 @@ static int cvSensEwtSetSS(CVodeMem cv_mem, N_Vector *yScur, N_Vector *weightS)
 
 static int cvSensEwtSetSV(CVodeMem cv_mem, N_Vector *yScur, N_Vector *weightS)
 {
+  SUNAssignSUNCTX(CV_SUNCTX);
   int is;
 
   for (is=0; is<cv_mem->cv_Ns; is++) {
-    N_VAbs(yScur[is], cv_mem->cv_tempv);
+    SUNCheckCallLastErrNoRet(N_VAbs(yScur[is], cv_mem->cv_tempv));
     N_VLinearSum(cv_mem->cv_reltolS, cv_mem->cv_tempv, ONE,
                  cv_mem->cv_VabstolS[is], cv_mem->cv_tempv);
     if (cv_mem->cv_atolSmin0[is]) {
       if (N_VMin(cv_mem->cv_tempv) <= ZERO) return(-1);
```

Comment:
As above, needs a `sunrealtype minval` declared at the top of the function, and then this should be changed to something like
```suggestion
    minval = SUNCheckCallLastErrNoRet(N_VMin(cv_mem->cv_tempv));
    if (minval <= ZERO) return(-1);
```